src/main/flight/pid.c

```diff
@@ -865,7 +861,8 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
 
     // gradually scale back integration when above windup point
     float dynCi = dT * itermAccelerator;
-    if (ITermWindupPointInv > 0) {
+    if (ITermWindupPointInv > 1.0f) {
+        const float motorMixRange = getMotorMixRange();
         dynCi *= constrainf((1.0f - motorMixRange) * ITermWindupPointInv, 0.0f, 1.0f);
```

Comment:
You can also directly use getMotorMixRange now
`dynCi *= constrainf((1.0f - getMotorMixRange()) * ITermWindupPointInv, 0.0f, 1.0f);`